catalog/ui/catalog-ui-search/src/main/webapp/component/ingest-details/ingest-details.view.js

```diff
@@ -38,6 +39,9 @@ function updateDropzoneHeight(view) {
 
 module.exports = Marionette.LayoutView.extend({
     template: template,
+    extraHeaders: {},
```

Comment:
No need to store this off, we can use this.options.[name] to access them where they're needed in the view.

Reply:
changed